src/converse-service-administration.js

```diff
@@ -0,0 +1,412 @@
+     * overridden or relied upon, and therefore need to be loaded before
+     * this plugin.
+     *
+     * If the setting "strict_plugin_dependencies" is set to true,
+     * an error will be raised if the plugin is not found. By default it's
+     * false, which means these plugins are only loaded opportunistically.
+     *
+     * NB: These plugins need to have already been loaded via require.js.
+     */
+    dependencies: ["converse-chatboxes", "converse-rosterview", "converse-chatview"],
+
+    initialize () {
+        const { _converse } = this,
+            { __ } = _converse;
+
+        _converse.SERVICE_ADMIN_TYPE = 'service-administration';
+
+        _converse.ServiceAdministration = Backbone.Model.extend({
+            async discoverSupport () {
+                const supported = await _converse.api.disco.supports(Strophe.NS.COMMANDS, _converse.bare_jid);
+                if (supported) {
+                    const stanza = $iq({
+                        'from': _converse.connection.jid,
+                        'id': u.getUniqueId(),
+                        'to': _converse.domain,
+                        'type': 'get'
+                    }).c('query', {
+                        xmlns: Strophe.NS.DISCO_ITEMS,
+                        node: 'announce'
+                    }).up();
+
+                    _converse.connection.sendIQ(stanza, result => {
+                        const commands = sizzle('item[node^="http://jabber.org/protocol/admin"]', result);
+                        _converse.serviceAdminCommands = new _converse.ServiceAdminCommandCollection();
+                        commands.forEach(command => {
+                            const command_description = command.getAttribute('name');
+                            const command_node = command.getAttribute('node');
+                            const command_name = command_node.split('#').pop();
+
+                            _converse.serviceAdminCommands.add(new _converse.ServiceAdminCommandItem({
+                                command_node: command_node,
+                                description: command_description,
+                                command_name: command_name
+                            }));
+                        });
+                        this.renderControlboxElement();
+                    });
+                }
+            },
+
+            renderControlboxElement () {
+                _converse.serviceAdministrationCommandView = new _converse.ServiceAdminCommandView({collection: _converse.serviceAdminCommands});
+                const groupchat_element = _converse.chatboxviews.get('controlbox').el.querySelector('#chatrooms');
+                if (groupchat_element !== null) {
+                    groupchat_element.insertAdjacentElement('beforebegin', _converse.serviceAdministrationCommandView.render());
+                }
+            }
+        });
+
+        _converse.ServiceAdminCommandItem = Backbone.Model.extend({});
```

Comment:
You don't have to call `.extend({})`, simply setting it to `Backbone.Model` is enough.